package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "devicon",
-  "version": "2.15.0",
+  "version": "2.15.1",
```

Comment:
This file shouldn't be edited for icon requests. Please revert/reset the changes